src/components/shared/Dashboard/DashboardNewDeploy.tsx

```diff
@@ -0,0 +1,80 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { Trans } from '@lingui/macro'
+import { Button } from 'antd'
+import FeedbackPromptModal from 'components/v1/V1Project/modals/FeedbackPromptModal'
+
+import { useState } from 'react'
+
+import { useHistory } from 'react-router-dom'
+
+import { layouts } from 'constants/styles/layouts'
+import { padding } from 'constants/styles/padding'
+
+export default function DashboardNewDeploy({
```

Comment:
**Suggestion:** I don't think this name reflects what this component really does, perhaps something more like 'NewDeployNotAvailable'